File: src/BukkitPE/event/level/LevelEvent.php
<?php
/**
 * Level related events
 */
namespace BukkitPE\event\level;

use BukkitPE\event\Event;
use BukkitPE\level\Level;

abstract class LevelEvent extends Event{
	/** @var \BukkitPE\level\Level */
	private $level;

	/**
	 * @param Level $level
	 */
	public function __construct(Level $level){
		$this->level = $level;
	}

	/**
	 * @return \BukkitPE\level\Level
	 */
	public function getLevel(){
		return $this->level;
	}
}

File: src/main/java/net/BukkitPE/event/level/LevelEvent.java
package net.BukkitPE.event.level;

import net.BukkitPE.event.Event;
import net.BukkitPE.level.Level;

/**

 * BukkitPE Project
 */
public abstract class LevelEvent extends Event {

    private Level level;

    public LevelEvent(Level level) {
        this.level = level;
    }

    public Level getLevel() {
        return level;
    }
}
